{
  "bad": false,
  "new_version": "4.0.3",
  "new_version_attempts": {
    "3.3.6": 1,
    "4.0.0": 1,
    "4.0.1": 1,
    "4.0.2": 1,
    "4.0.3": 3
  },
  "new_version_errors": {
    "4.0.3": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/11938627452\">bot CI job</a>): main:\nError running 'conda-forge-tick-container migrate-feedstock --feedstock-name jsonrpcclient --default-branch main --existing-feedstock-node-attrs - --log-level info --kwargs {\n \"hash_type\": \"sha256\"\n}' in container - error VersionMigrationError raised:\n\"The recipe did not change in the version migration, a URL did not hash, or there is jinja2 syntax the bot cannot handle!\n\nPlease check the URLs in your recipe with version '4.0.3' to make sure they exist!\n\nWe also found the following errors:\n\n - could not hash URL template 'https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz'\n\""
  }
}